packages/holocron/src/HolocronErrorBoundary.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { node } from 'prop-types';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+
+class HolocronErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: null };
+  }
+
+  static getDerivedStateFromError(error) {
+    return { error };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({
+      error,
+    });
```

Comment:
this isn't needed with `getDerivedStateFromError`

Reply:
@10xLaCroixDrinker are you fine if I add `// eslint-disable-next-line class-methods-use-this` above the `componentDidCatch(error, errorInfo)` then?

Reply:
Taken care of in b6e3011